keystores/src/main/java/com/palantir/conjure/java/config/ssl/SslSocketFactories.java

```diff
@@ -269,6 +311,23 @@ public static KeyManager[] createKeyManagers(SslConfiguration config) {
 
     private static TrustManagerFactory createTrustManagerFactory(
             Path trustStorePath, SslConfiguration.StoreType trustStoreType) {
+        KeyStore keyStore = getCombinedTrustStoreAndDefaultCas(trustStorePath, trustStoreType);
+
+        return getTrustManagerFactory(keyStore);
+    }
+
+    private static TrustManagerFactory getTrustManagerFactory(KeyStore keyStore) {
+        try {
+            TrustManagerFactory trustManagerFactory =
+                    TrustManagerFactory.getInstance(TrustManagerFactory.getDefaultAlgorithm());
+            trustManagerFactory.init(keyStore);
+            return trustManagerFactory;
+        } catch (GeneralSecurityException e) {
+            throw Throwables.propagate(e);
+        }
+    }
+
+    private static KeyStore getCombinedTrustStoreAndDefaultCas(Path trustStorePath, StoreType trustStoreType) {
```

Comment:
This would be convenient if we made it public, this PR would not be necessary if this method was available.